Product/SoapUI_Test/RegressionSuite/pom.xml

```diff
@@ -29,6 +29,7 @@
                 <module>Passthru</module>
                 <module>Bimodal</module>
                 <module>NewFeature</module>
+				<module>Manual</module>
```

Comment:
If you include this, it will hit manual when we invoke -Dregression.  Suggest to move into their own profile